dokka-subprojects/analysis-java-psi/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ plugins {
 
 dependencies {
     compileOnly(projects.dokkaSubprojects.dokkaCore)
+    compileOnly(projects.dokkaSubprojects.analysisKotlinApi)
```

Comment:
Is it really a good idea to make java-psi and markdown analysis depend on kotlin-api?
For me it looks really strange...
May be it's better to move this declaration to `core`, in `internal` package? As we will anyway do something later with it?